en/services/ftp.md

```diff
@@ -446,7 +446,7 @@ The implementation on PX4 only supports a single session.
 
 ## CRC32 Implementation
 
-The CRC32 calculation used by MAVLink FTP is similar to the ISO 3309 standard based on the polygon 0x04C11DB7. This is also referred to the CRC32 based on Gary Brown's work.
+The CRC32 calculation used by MAVLink FTP is similar to but different from the ISO 3309 standard based on the polygon 0x04C11DB7, which is also referred to the CRC32 based on Gary Brown's work.
```

Comment:
@julianoes I don't understand this -  it isn't clear what is "based on Gary Brown's work". I THINK you are saying that Gary Brown refers to our particular implementation as a CRC32? Or you could mean that our implementation is based on Gar Browns work. 

My guess

> The CRC32 calculation used by MAVLink FTP is similar to (but different from) the ISO 3309 standard based on the polygon 0x04C11DB7. The implementation is based on Gary Brown's work, and is referred to as the "CRC32" in our documentation.

Reply:
The CRC algorithm that we are using is commonly referred to as Gary Brown's work.
That's also what is written in some of the source files, e.g. NuttX.

Reply:
Thanks. For your review in https://github.com/mavlink/mavlink-devguide/pull/300